api/StaticRange.json

```diff
@@ -54,16 +54,16 @@
           "description": "<code>StaticRange()</code> constructor",
           "support": {
             "chrome": {
-              "version_added": "60"
+              "version_added": false
             },
             "chrome_android": {
-              "version_added": "60"
+              "version_added": false
             },
             "edge": {
-              "version_added": "18"
+              "version_added": null
```

Comment:
> After some further review, it turns out that Edge **does** actually have support since version 18. 

If you run the test at http://web-platform-tests.live/dom/ranges/StaticRange-constructor.html in Edge, does it pass?

Reply:
> If you run the test at http://web-platform-tests.live/dom/ranges/StaticRange-constructor.html in Edge, does it pass?

Or http://w3c-test.org/dom/ranges/StaticRange-constructor.html

Reply:
Can't access that first link, but the second link shows that it's not working.  Sorry, never mind!  Let's set it to `false` then.

Reply:
> Can't access that first link, but the second link shows that it's not working.

Yeah, they recently changed the domain name (but neglected to set up redirect from the old one):

http://wpt.live/dom/ranges/StaticRange-constructor.html

> Sorry, never mind! Let's set it to `false` then.

OK